### PR TITLE
Update charge message to include year

### PR DESCRIPTION
**Pull request by Tiffanylin21**
Update according to [PE-D] issue #138: Lack of year input check for CHARGE and according to [PE-D] issue #132: Hard to ascertain if charge calculation is correct

### CODE REVIEW

**Comment by Dineshraj555** [comment]
Easy and simple fix! Nice Work!

File: src/main/java/seedu/address/logic/commands/ChargeCommand.java [hunk 1]
```diff
@@ -81,7 +83,39 @@ public CommandResult execute(Model model) throws CommandException {
...
+            if (hasAttendance) {
+                AttendanceEntry entry = map.getAttendance(day).get();
+                boolean isPresent = entry.getIsPresent().orElse(false);
+                if (isPresent) {
+                    amountChargeable += this.charge.getCharge();
+                    attendance += "\n" + entry.getAttendanceDate().format(formatter);
+                }
+            }
+            day = day.plusDays(1);
+        }
```
Comment (tobihy) [inline src/main/java/seedu/address/logic/commands/ChargeCommand.java:111]: I think this part is a bit too nested! Could benefit from more refactoring, depending on how you would like to organise your code.

File: src/main/java/seedu/address/logic/commands/ChargeCommand.java [hunk 2]
```diff
@@ -81,7 +83,39 @@ public CommandResult execute(Model model) throws CommandException {
...
+        if (amountChargeable == 0) {
+            attendance = "\n" + pet.getName().toString() + " was not present on any day.";
+        }
+        if (amountChargeable == this.charge.getCharge()) {
+            attendance = attendance.replace("are the days", "is the day");
+        }
```
Comment (tobihy) [inline src/main/java/seedu/address/logic/commands/ChargeCommand.java:117]: For the string constants, perhaps it is better to declare them as `private static final` at the start and use `String.format()` on the String for better code organisation?

**Review by tobihy — COMMENTED** [review]
Some minor nits to be looked into. Other than that, good work!

File: src/main/java/seedu/address/logic/commands/ChargeCommand.java (hunk 2 above)
Comment (Tiffanylin21) [reply src/main/java/seedu/address/logic/commands/ChargeCommand.java]: True, I shall correct this!

File: src/main/java/seedu/address/logic/commands/ChargeCommand.java (hunk 1 above)
Comment (Tiffanylin21) [reply src/main/java/seedu/address/logic/commands/ChargeCommand.java]: good suggestion, I think I will extract the inner-if statement to a boolean check to reduce nesting!

**Pull request merged**